make/brogue.mk

```
@@ -1,5 +1,8 @@
 bin/brogue$(.exe): $(objects) vars/cflags vars/LDFLAGS vars/libs vars/objects make/brogue.mk
 	$(CC) $(cflags) $(LDFLAGS) -o $@ $(objects) $(libs)
+# on windows, embedding the manifest modifies the executable, preventing debugging
 ifeq ($(SYSTEM),WINDOWS)
+  ifeq ($(DEBUG),NO)
 	mt -manifest windows/brogue.exe.manifest '-outputresource:bin/brogue.exe;1'
+  endif
```

as per c9be0cf09f506c9cced3e9edc2d2659d31c47b8d, I think some Makes are fussy with indentation. Also saves mixing tabs+spaces

```suggestion
ifeq ($(DEBUG),NO)
	mt -manifest windows/brogue.exe.manifest '-outputresource:bin/brogue.exe;1'
endif
```

---

Yeah for me it only works if spaces are used, not tab. I had originally tried using tabs and it failed my tests. Here's a snippet from the gnu make docs that explains: https://www.gnu.org/software/make/manual/html_node/Conditional-Syntax.html

> Extra spaces are allowed and ignored at the beginning of the conditional directive line, but a tab is not allowed. (If the line begins with a tab, it will be considered part of a recipe for a rule.) Aside from this, extra spaces or tabs may be inserted with no effect anywhere except within the directive name or within an argument. A comment starting with ‘#’ may appear at the end of the line.

---

Oh, wait. You're not suggesting using tab, just eliminating the indent. That should work. I'll try it.